mlrun/db/httpdb.py

```diff
@@ -405,6 +414,15 @@ def store_log(self, uid, project="", body=None, append=False):
         error = f"store log {project}/{uid}"
         self.api_call("POST", path, error, params, body)
 
+    def stream_log(self, uid: str, project: str = ""):
+        """Stream a log from the API
+        :param uid: Log unique ID
+        :param project: Project name for which this log belongs
+        """
```

Comment:
I would add a TODO and a jira ticket for us to change the client to use the `stream_log` and get rid from our current implementation

Reply:
Well, I think I'll remove it altogether. `get_log` uses stream anyways